charts/s3-data/values.yaml

```diff
@@ -4,13 +4,13 @@
 replicaCount: 1
 image:
   repository: zenko/cloudserver
-  tag: pensieve-0
+  tag: 0.1.9
   pullPolicy: IfNotPresent
 service:
   name: s3-data
   type: ClusterIP
-  internalPort: 9992
-  externalPort: 9992
+  internalPort: 9991
+  externalPort: 9991
```

Comment:
If this changes, I would kind-of expect a related change somewhere else? Or is 9991 hard-coded in some other service? If that's the case, why have it configurable here?

`internalPort` shouldn't have been configurable from the start, actually. That's something we should fix.